.github/workflows/continuous-deployment.yml

```
@@ -1,9 +1,10 @@
+---
 name: "Continuous Deployment"
 
-on:
-  workflow_dispatch:
+"on":
+  workflow_dispatch: true
```

actionlint (1.7.7)

"workflow_dispatch" section is scalar node but mapping node is expected

(syntax-check)